### PR TITLE
[Pricing Block] fixing use case with no plan authored

### PR DESCRIPTION
**Pull request by qiyundai**
Fixing a broken old pricing block on live pages

Resolves: [MWPW-NUMBER](https://jira.corp.adobe.com/browse/MWPW-NUMBER) pending

Test URLs:
- Before: https://main--express--adobecom.hlx.page/express/spotlight/flyers?martech=off
- After: https://pricing-block-fix--express--adobecom.hlx.page/express/spotlight/flyers?martech=off

### CODE REVIEW

File: express/blocks/pricing/pricing.js [hunk 1]
```diff
@@ -100,9 +99,9 @@ function decorateIconList($pricingRight) {
   if ($iconList.children.length > 0) $pricingRight.appendChild($iconList);
 }
 
-function selectPlan($block, plan) {
-  const $title = $block.querySelector('.pricing-plan-title');
-  const $dropdown = $block.querySelector('.pricing-plan-dropdown');
+function selectPlan(block, plan) {
+  const $title = block.querySelector('.pricing-plan-title');
```
Comment (redvelvet511) [inline express/blocks/pricing/pricing.js:103]: If we're keeping the pattern of omitting the $ sign, should we remove it in $title and $dropdown too?

File: express/blocks/pricing/pricing.js [hunk 2]
```diff
@@ -128,19 +127,19 @@ async function selectPlanOption($block, plan, planOption) {
...
+  const planDivs = Array.from(contentEls.children);
   let plan;
   let planId = 0;
   let planOptionId = 0;
-  $planDivs.forEach(($rowContent) => {
-    const $rowContents = Array.from($rowContent.children);
-    $rowContents.forEach(($content) => {
+  planDivs.forEach((rowContent) => {
+    const rowContents = Array.from(rowContent.children);
+    rowContents.forEach(($content) => {
```
Comment (redvelvet511) [inline express/blocks/pricing/pricing.js:142]: also here, $content

File: express/blocks/pricing/pricing.js [hunk 3]
```diff
@@ -191,130 +191,134 @@ function populateOtherPlans($contents) {
   return otherPlans;
 }
 
-function closeActivePopups($block, $except) {
-  $block.querySelectorAll(':scope .active').forEach(($activePopup) => {
-    if ($activePopup !== $except) {
-      $activePopup.classList.remove('active');
+function closeActivePopups(block, $except) {
```
Comment (redvelvet511) [inline express/blocks/pricing/pricing.js:194]: $except

**Review by redvelvet511 — COMMENTED** [review]
Just the pattern of omitting the $ sign if needed to keep a constant pattern.
There are 4 words left.

**Review by JingleH — APPROVED** [review]
I'm agreeing with @redvelvet511 that we can remove those dollar signs by the way, but otherwise this fix is good

**Pull request merged**